### PR TITLE
Except non-static commands from the broadcast cooldown

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

**Comment by Zarel** [comment]
This seems to be the entirely wrong solution? _Most_ commands should not be repeated, and repetition is accidental, e.g. if someone asks for a password reset, only one `!helpticket` should go through.

I would rather just whitelist the commands like `!roll` that _can_ be repeated.

**Comment by whalemer** [comment]
bump, does this way work?

File: chat-plugins/info.js [hunk 1]
```diff
@@ -640,7 +640,7 @@ exports.commands = {
 	resist: 'weakness',
 	weakness: function (target, room, user) {
 		if (!target) return this.parse('/help weakness');
-		if (!this.runBroadcast()) return;
+		if (!this.runBroadcast(true)) return;
```
[the remark this answers is not in the document]
Comment (whalemer) [reply chat-plugins/info.js]: yeah my bad, I didn't realise that before, I'll update soon

**Comment by whalemer** [comment]
updated
whitelisted commands are:
roll, pick, code, htmlbox, rand*, hangman display, poll display (they're uhtml), checkchallenges, evals

**Comment by Zarel** [comment]
I would remove `hangman display` and `poll display` from that list. Unless you mean something weird by "they're uhtml"?

**Comment by whalemer** [comment]
fair enough, updated

**Comment by Zarel** [comment]
Cool, I think this is good now.

**Pull request merged**